src/libs/actions/Report.js

```diff
@@ -649,6 +650,33 @@ function handleReportChanged(report) {
     reportMaxSequenceNumbers[report.reportID] = report.maxSequenceNumber;
 }
 
+/**
+ * Resets the Attachment Modal Data to default values.
+ */
+function clearAttachmentModalData() {
+    Onyx.set(ONYXKEYS.ATTACHMENT_MODAL, {
+        sourceURL: null, isAttachment: null, isModalOpen: false, file: null,
+    });
+}
+
+/**
+ * Sets the data for the attachment modal.
+ *
+ * @param {Object} reportID
+ * @param {String} reportID.sourceURL
+ * @param {Boolean} reportID.isAttachment
+ * @param {Boolean} reportID.isModalOpen
+ * @param {Object} reportID.file
+ * @param {String} reportID.file.name
```

Comment:
unused param

Reply:
Im confused, file defaults to null, but it can also be an object with a name attribute

Reply:
True its weird that file defaults to null. But even either way `reportID.file` is Object so its param type is correct. We don't mention child params in JSDocs unless its' explicitly used by the method and since this method doesn't directly use `reportID.file.name` we can remove its mention here. Plus some editors will even complain about it,
![image](https://user-images.githubusercontent.com/14356145/104511108-19731480-55a1-11eb-92e1-b8c751592c89.png)

Reply:
ok